pkgs/top-level/all-packages.nix

```diff
@@ -24067,6 +24067,8 @@ with pkgs;
   nv-codec-headers-11 = callPackage ../development/libraries/nv-codec-headers/11_x.nix { };
   nv-codec-headers-12 = callPackage ../development/libraries/nv-codec-headers/12_x.nix { };
 
+  nvidia-container-toolkit = callPackage ../applications/virtualization/nvidia-container-toolkit { };
+
   nvidiaCtkPackages =
     callPackage ../applications/virtualization/nvidia-container-toolkit/packages.nix
       { };
```

Comment:
1. Why not `self.callPackage` inside the `nvidiaCtkPackages` scope and `inherit` in `all-packages.nix`?
2. Do we still need the scope if you can instantiate the `nvidia-container-toolkit` independently? of the docker/podman stuff? (I suppose yes because we don't remove the wrappers yet?)
3. IMO we do need to get Ofborg green

Reply:
1. Can you be more explicit about your request? There's packages.nix and default.nix, I just want to add default.nix that provides nvidia-container-toolkit
2. Yes, we still need it because that's what provides nvidia-docker and nvidia-podman. We will clean up, but not yet.
3. I find it weird to move nvidia-container-toolkit while I just need to expose it in order to consume, it was already in `applications/virtualization`.

Reply:
Moved to by-name

Reply:
1. Move your `nvidia-container-toolkit = callpackage .. { }` to above https://github.com/NixOS/nixpkgs/blob/28d6a724f54085377102db7c3278ba82a0a5255f/pkgs/applications/virtualization/nvidia-container-toolkit/packages.nix#L32-L35
   Add `inherit (nvidiaCtkPackages) nvidia-container-toolkit` to `all-packages.nix`

Reply:
> Moved to by-name

Or maybe we stick with that, I'll have a look at the diff later